packages/truffle-core/lib/build.js

```diff
@@ -0,0 +1,133 @@
+const async = require("async");
+const mkdirp = require("mkdirp");
+const del = require("del");
+const fs = require("fs");
+const Contracts = require("truffle-workflow-compile");
+const BuildError = require("./errors/builderror");
+const child_process = require("child_process");
```

Comment:
should this be childProcess for consistency?

Reply:
The only usage of it is `spawn`, and in the official [documentation](https://nodejs.org/api/child_process.html), they deconstruct directly to it. So we should probably do the same. For example:

```js
const { spawn } = require('child_process');
```

Reply:
It's annoying that this is what Node calls it. I usually sidestep the issue by requiring specific functions, not all of `child_process`.

Reply:
@adrianmcli Yes, I like deconstructing it.